spec/helpers/administrate/application_helper_spec.rb

```diff
@@ -41,4 +41,43 @@
       end
     end
   end
+
+  describe "#svg_tag" do
+    subject { Nokogiri::HTML svg_tag('logo.svg',
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.